--- fontconfig-2.8.0.orig/fc-lang/Makefile.am	2009-11-18 09:27:43.000000000 -0500
+++ fontconfig-2.8.0/fc-lang/Makefile.am	2013-02-08 12:51:38.193732689 -0500
@@ -24,9 +24,11 @@
 
 CC = @CC_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
-LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
+LINK = $(CCLD) $(AM_CFLAGS) $(BUILD_CFLAGS) $(AM_LDFLAGS) $(BUILD_LDFLAGS) -o $@
 
 INCLUDES=-I${top_srcdir}/src -I${top_srcdir} $(WARN_CFLAGS)
+COMPILE = $(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
+	$(BUILD_CPPFLAGS) $(AM_CFLAGS) $(BUILD_CFLAGS)
 
 TMPL=fclang.tmpl.h
 STMPL=${top_srcdir}/fc-lang/fclang.tmpl.h
