--- /dev/null	2013-01-11 10:03:01.932277919 -0500
+++ glib-2.33.2/glib/klaatu-stub.c	2013-02-11 19:02:04.353732600 -0500
@@ -0,0 +1,31 @@
+#include "config.h"
+
+#include <stdarg.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <locale.h>
+
+
+
+
+
+/* Return the numeric/monetary information for the current locale.  */
+extern struct lconv *localeconv (void) {
+  fprintf(stderr,"localeconv stub called\n");
+#if 0
+  // I may need to fill in a semi real one. we'll see.
+  if (!MyLocaleStub){
+    MyLocaleStub = malloc(siezof(struct lconv));
+  }
+  return MyLocaleStub;
+#endif
+  return NULL;
+
+
+}
+
+void setpwent(void)
+{
+  fprintf(stderr,"setpwent stub called\n");
+}
+
